alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
```

Comment:
For now I would suggest just using `$HOME`.

We should use the simplest possible expression unless we run into a situation where it could be ambiguous. I don't see any way that can happen in these expressions.